webpack/prod.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const TerserPlugin = require('terser-webpack-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const common = require('./common.config.js');
+
+const mode = 'production';
+
+module.exports = merge(common(mode), {
+  mode,
+  optimization: {
+    minimizer: [new TerserPlugin(), new OptimizeCSSAssetsPlugin({})]
+  },
+  externals: {
+    // Don't bundle react or react-dom
+    react: {
+        commonjs: "react",
+        commonjs2: "react",
+        amd: "React",
+        root: "React"
+    },
+    "react-dom": {
```

Comment:
ah, this is great. We don't need to do the same for slate though right...?

Reply:
hmm i'm not sure. Will test this in first and see if we can exclude slate as well